src/py/flwr/common/record/metricsrecord.py

```diff
@@ -84,3 +84,15 @@ def __init__(
                 self[k] = metrics_dict[k]
                 if not keep_input:
                     del metrics_dict[k]
+
+    def count_bytes(self) -> int:
+        """Return number of Bytes stored in this object."""
+        num_bytes = 0
+
+        for v in self.values():
+            if isinstance(v, List):
+                # both int and float take 4 bytes
+                num_bytes += 4 * len(v)
+            else:
+                num_bytes += 4
```

Comment:
Same as above, I think we should use `8` instead of `4`. Plus, we should count the bytes of keys.